packages/fx-core/src/component/driver/m365Bot/interface/updateM365BotArgs.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface UpdateM365BotArgs {
+  botId: string;
+  messagingEndpoint: string;
+  name?: string;
```

Comment:
Just for m365bot actions, prefer to keep the same args for both `create` and `update`.
Or, reconsider to only one action `m365Bot/createOrUpdate` ?

Reply:
I just realized that user may get confused if they can customize `name` in both `create` and `update`. So one single action seems clearer to user.

Reply:
Agree to use only one action `m365Bot/createOrUpdate`.